doc/classes/LineEdit.xml

```diff
@@ -316,6 +333,9 @@
 			[b]Note:[/b] This method is only implemented on Linux.
 		</member>
 		<member name="mouse_default_cursor_shape" type="int" setter="set_default_cursor_shape" getter="get_default_cursor_shape" overrides="Control" enum="Control.CursorShape" default="1" />
+		<member name="overtype_mode_enabled" type="bool" setter="set_overtype_mode_enabled" getter="is_overtype_mode_enabled" default="false">
+			If [code]true[/code], sets the user into overtype mode. When the user types in this mode, it will override existing text.
```

Comment:
Copied straight from a revamped **TextEdit** description.
```suggestion
			If [code]true[/code], enables overtype mode. In this mode, typing overrides existing text instead of inserting text. The [member ProjectSettings.input/ui_text_toggle_insert_mode] action toggles overtype mode.
```